modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -0,0 +1,116 @@
+= Policy Level
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies implement an additional level of granularity with SLA tiering. Policies control access based on the following access structuring:
+
+* Resource-level policies
+Impose a limit on all requests that match the resource to which it is applied. All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
+* SLA-based policies
+Impose a limit specified by the level of access granted to the requesting application. The Rate Limiting policy is SLA-based.
+
+When the API Gateway is configured to work as a Mule Cluster, the policy can be configured to work in a distributed manner.
+  
+  [PA: This seems out of place and does not make sense.]
+
+[PA: We need to tell the customers how to work with SLA-tiers for policies right?]
```

Comment:
SLA tiers for me should go somewhere else and link to all the documentation on contracts, request api access and those flows.

Reply:
Made a note for our discussion! Thanks!

Reply:
Done! Thanks!